### PR TITLE
nf-core: add Page

### PR DESCRIPTION
**Pull request by apeltzer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/nf-core.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`nf-core create`
+
+- Lint the pipeline code:
+
+`nf-core lint {path}`
+
+- Bump software versions in pipeline recipe:
+
+`nf-core bump-version`
```
Comment (ewels) [inline pages/common/nf-core.md]:
```suggestion
`nf-core bump-version {{/path/to/directory}} {{new_version}}`
```

File: pages/common/nf-core.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`nf-core list`
+
+- Create a new pipeline skeleton:
+
+`nf-core create`
+
+- Lint the pipeline code:
+
+`nf-core lint {path}`
```
Comment (ewels) [inline pages/common/nf-core.md]:
```suggestion
`nf-core lint {{/path/to/directory}}`
```

File: pages/common/nf-core.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Bump software versions in pipeline recipe:
+
+`nf-core bump-version`
+
+- Launch an existing pipeline (e.g. {pipelinename}):
```
Comment (ewels) [inline pages/common/nf-core.md]:
```suggestion
- Launch an nf-core pipeline:
```

File: pages/common/nf-core.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`nf-core launch {{pipelinename}}`
```
Comment (ewels) [inline pages/common/nf-core.md]:
```suggestion
`nf-core launch {{pipeline_name}}`
```

File: pages/common/nf-core.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download an existing pipeline  (e.g. {pipelinename}):
```
Comment (ewels) [inline pages/common/nf-core.md]:
```suggestion
- Download an nf-core pipeline for offline use:
```

File: pages/common/nf-core.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`nf-core download {{pipelinename}}`
```
Comment (ewels) [inline pages/common/nf-core.md]:
```suggestion
`nf-core download {{pipeline_name}}`
```

File: pages/common/nf-core.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`nf-core list`
+
+- Create a new pipeline skeleton:
+
+`nf-core create`
+
+- Lint the pipeline code:
+
+`nf-core lint {{/path/to/directory}}`
```
Comment (agnivade) [inline pages/common/nf-core.md]: Please remove the preceding `/`

File: pages/common/nf-core.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Bump software versions in pipeline recipe:
+
+`nf-core bump-version {{/path/to/directory}} {{new_version}}`
```
Comment (agnivade) [inline pages/common/nf-core.md]: Same as above.

Comment (apeltzer) [reply pages/common/nf-core.md]: Done!

File: pages/common/nf-core.md (hunk 7 above)
Comment (apeltzer) [reply pages/common/nf-core.md]: Done!

**Comment by apeltzer** [comment]
Thanks! Just added the required changes directly.

**Pull request merged**